front/app/components/SignUpIn/AuthProviderButton.tsx

```diff
@@ -167,7 +168,7 @@ const AuthProviderButton = memo<Props>(
     return (
       <Container className={className} id={id}>
         <Button
-          icon={authProvider as any}
+          icon={icon as any}
```

Comment:
The as any should be removed here. Instead we need to change the prop type of icon to IconNames.

Reply:
Where do I get the type `IconNames` from?

Reply:
The component library :-) 

![Screenshot 2022-02-24 163633](https://user-images.githubusercontent.com/16427929/155556261-6edc16a5-1ade-40b8-a6de-5fd1763a25d0.png)